File: HW1/Triangle.java
public class Triangle extends Shape {

	private int h, b; // h- height, b-base

	public Triangle (int nx, int ny, int nz, int nh, int nb) {
		super ("triangle", nx, ny, nz);
		h = nh;
		b = nb;
	}

	public void display() {
		super.display();
		System.out.println ("height : " + h + " base : " + b  + " ]"); 
	}

}

File: HW2/Triangle.py
from Shape import *

class Triangle (Shape):

  	def __init__(self, nx, ny, nz, nh, nb):
  		super(Triangle, self).__init__("triangle", nx, ny, nz)
  		self._h = nh
  		self._b = nb

  	def getArea(self):
  		return 0.5*self._h * self._b;

	def display(self):
		super(Triangle, self).display()
		print("[ height : " + str(self._h) + " base : " + str(self._b) + " ]")